superset_patchup/views.py

```diff
@@ -50,7 +50,7 @@ def all_dashboards(self):
         # See: https://github.com/apache/incubator-superset/blob/0.27/
         # superset/views/core.py#L154
         view_filter = \
-            superset.views.core.DashboardFilter(
+            superset.views.dashboard.filters.DashboardFilter(
```

Comment:
What are the implications of this kind of change on using ketchup for older versions of Superset?  Do we need to maintain a list showing superset versions and ketchup versions that are compatible?

Reply:
This version, actually a couple commits already in master cannot be deployed to old superset version. I think we should have a breaking change. If we backwards port we should create commits. and tag from tag releases of v0.1.6